tools/install/DynamoInstaller.iss

```diff
@@ -199,7 +199,7 @@ begin
     begin
 	  MsgBox('Dynamo requires an installation of Revit 2014, Revit 2015 or Revit 2016 in order to proceed!', mbCriticalError, MB_OK);
       result := false;
-    end
+    end;
```

Comment:
The latest `InnoSetup` version on my machine refused to parse this script without the semi-colon here.